rotkehlchen/inquirer.py

```diff
@@ -388,8 +440,84 @@ def find_usd_price(
             Inquirer._cached_current_price[cache_key] = CachedPriceEntry(price=price, time=ts_now())  # noqa: E501
             return price
 
+        if is_known_protocol or underlying_tokens is not None:
+            usd_price = get_underlying_asset_price(token)  # type: ignore
+            if usd_price is None:
+                price = Price(ZERO)
+            else:
+                price = Price(usd_price)
+            Inquirer._cached_current_price[cache_key] = CachedPriceEntry(
+                price=price,
+                time=ts_now(),
+            )
+            return price
+
         return instance._query_oracle_instances(from_asset=asset, to_asset=A_USD)
 
+    def find_uniswap_v2_lp_price(
+        self,
+        token: Union[EthereumToken, UnknownEthereumToken],
+    ) -> Optional[Price]:
+        """
+        First thing should be query oracles for prices. It it files then we
+        calculate price for a LP token. This one is
```

Comment:
```suggestion
        First thing should be query oracles for prices. It it files then we
        calculate price for a LP token. This one is
```
These 2 sentences don't make sense. What do you want to say? What this function does? So query oracles for prices of underlying LP assets?